926223 - net-snmp: Does not support aarch64 in f19 and rawhide

Update autoconf version to make the test suite happy.

diff -up net-snmp-5.7.2/dist/autoconf-version.autoreconf net-snmp-5.7.2/dist/autoconf-version
--- net-snmp-5.7.2/dist/autoconf-version.autoreconf	2013-03-25 13:00:15.002745347 +0100
+++ net-snmp-5.7.2/dist/autoconf-version	2013-03-25 13:00:17.207736442 +0100
@@ -1 +1 @@
-2.68
+2.69
